src/components/PostList/PostList.jsx

```diff
@@ -16,6 +16,7 @@ const mockPostList = [
       title: 'Location title',
       link: 'https://googole.com',
     },
+    postPicture: 'https://via.placeholder.com/614x420',
```

Comment:
Will better to use the next format instead of `postPicture ` : 
```
assets: [
    {
        type: 'IMAGE',
        src: 'link on resource' 
    }
 ]
```

With this format we can cover galleries and videos.